tests/conftest.py

```diff
@@ -72,7 +72,7 @@ async def mock_get_properties(dbus_obj, interface):
         if latest.isnumeric():
             fixture = f"{fixture}_{latest}"
 
-        return load_json_fixture(f"{fixture}.json")
+        return load_json_fixture(f"{fixture or 'base'}.json")
```

Comment:
Why we need that?

Reply:
https://github.com/home-assistant/supervisor/pull/3136/files#diff-e891abd55fca9889061fdb3c890811f14179b0e4463bc0dbbd78e3f5a362091dR119
This is `""`
Which would look for  the file `fixtures/.json`

Reply:
Not without patching this helper in this single test

Reply:
We don't require 100% coverage, so can just remove the testing parts out of the PR :shrug: